README.md

```diff
@@ -82,6 +82,13 @@ dotnet run skip:dotnet run skip:PackageNetCoreV2Bundle,PackageNetCoreV3BundlesWi
     - https://github.com/Azure/azure-functions-templates#testing-script-type-template-via-core-tools
 
 
+## Debugging in Visual Studio

```

Comment:
Would change the title to `Debugging build` or `Debugging bundles build` otherwise someone might misunderstand it as debugging bundles itself.